packages/client/src/lib/interface.ts

```diff
@@ -1,6 +1,8 @@
 import type { UnixFSEntry } from 'ipfs-car/unpack'
 import type { CID } from 'multiformats'
 export type { CID, UnixFSEntry }
+import type { MemoryBlockStore as Blockstore } from 'ipfs-car/blockstore/memory'
```

Comment:
This doesn't have to be a `MemoryBlockStore` does it?

```suggestion
import type { Blockstore } from 'ipfs-car/blockstore'
```